packages/create-remix/index.ts

```diff
@@ -317,7 +317,9 @@ async function copyTempDirToAppDirStep(ctx: Context) {
 
   let files1 = await getDirectoryFilesRecursive(ctx.tempDir);
   let files2 = await getDirectoryFilesRecursive(ctx.cwd);
-  let collisions = files1.filter((f) => files2.includes(f));
+  let collisions = files1
+    .filter((f) => files2.includes(f))
+    .sort((a, b) => a.localeCompare(b));
```

Comment:
I ran into a test failure due to these files being printed to the console in a different order, so sort the collisions list for stability